addons/mrp_subcontracting/static/src/components/bom_overview_components_block/mrp_bom_overview_components_block.js

```diff
@@ -1,4 +1,4 @@
-/** @odoo-module **/
+/* @odoo-module **/
```

Comment:
:confused: :thinking: 

I guess this change was a mistake, but I'm also pretty sure you don't need to add `/** @odoo-module **/` to new JS Components as of March 7 (from the guru meeting "_in master, the /* @odoo-module */ comment on top of js files is no longer required (static/src/ and static/tests/ files are now considered module by default)_". We just haven't gotten around to removing them yet from the logistics code :crying_cat_face:

Reply:
this was of course a mistake :) I've just fixed it